homeassistant/components/thermostat/homematic.py

```diff
@@ -108,21 +144,48 @@ def device_state_attributes(self):
                 "battery": self._battery,
                 "mode": self._mode}
 
+    @property
+    def min_temp(self):
```

Comment:
That sounds more like a heater/cooler to me instead of a thermostat. Is this the case for all homematic thermostats? And wouldn't it be better for those devices to be represented as a switch platform ?